### PR TITLE
Add recipe for ubermagtable.

### PR DESCRIPTION
**Pull request by marijanbeg**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/ubermagtable/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{% set version = "0.1.2" %}
+{% set name = "ubermagtable" %}
+{% set sha256 = "62c8920d08b1ac15da371c2df11293dcbefdd867fbf299f6fa9dd67fb353f110" %}
+
+package:
+  name: {{ name | lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
```
Comment (chrisburr) [inline recipes/ubermagtable/meta.yaml]: This line isn't needed
```suggestion
```

File: recipes/ubermagtable/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+test:
+  imports:
+    - ubermagtable
+  commands:
+    - python -c "import sys; import ubermagtable as m; sys.exit(m.test())"
+
+about:
+  home: http://ubermag.github.io/
+  license: BSD 3-clause
```
Comment (chrisburr) [inline recipes/ubermagtable/meta.yaml]: For consistency with other recipes, the SPDX identifier has a dash:
```suggestion
  license: BSD-3-clause
```

File: recipes/ubermagtable/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: pip install --no-deps .
```
Comment (chrisburr) [inline recipes/ubermagtable/meta.yaml]: This sometimes makes debugging easier:
```suggestion
  script: pip install --no-deps . -vvv
```

**Comment by marijanbeg** [comment]
@conda-forge-admin, please rerender

**Comment by marijanbeg** [comment]
@conda-forge-admin, please rerender

**Comment by marijanbeg** [comment]
Thank you @chrisburr. Is there anything else I should do?

**Pull request merged**